src/services/localSSHService.ts

```diff
@@ -44,12 +48,31 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
                     // event, so ignore it if more settings are affected at the same time.
                     return;
                 }
-                if (this.initialized) {
-                    await this.initialized;
-                }
-                this.initialized = this.initialize();
+                await this.initWhenFlagEnabled();
             }
         }));
+        this.experimentService.onDidChangeFlag(async () => {
+            await this.initWhenFlagEnabled();
```

Comment:
It will not work because then setting changed, there is no `userId` so I need to wait until we have userId here, but not sure if it's good. cc @jeanp413 

```
[warning] ConfigCat: Evaluating getValue('gitpod_desktop_use_local_ssh_proxy'). UserObject missing! You should pass a UserObject to getValue(), in order to make targeting work properly. Read more: https://configcat.com/docs/advanced/user-object
```